### DISCUSSION TITLE
How to migrate bindingElementExtensions and behaviorExtensions in my Web.config to CoreWCF

### DISCUSSION THREAD

**Discussion by alexvazquez** [CoreWCF/CoreWCF#1076] [General]
I have in my .NET Framework WCF Services Web.Config the following extensions:
```

<extensions>
      <bindingElementExtensions>
        <add name="gzipMessageEncoding" type="App.Framework.Compression.GZipMessageEncodingElement, App.Framework.Compression, Version=2.0.2.5, Culture=neutral, PublicKeyToken=c82ac3487be12f4b"/>
      </bindingElementExtensions>
      <behaviorExtensions>
        <add name="MessageInspectorBehavior" type="App.Framework.WebServices.WCFBehavoirs.ServerInspectorBehavoirElement, App.Framework.WebServices, Version=2.0.2.5, Culture=neutral, PublicKeyToken=null"/>
      </behaviorExtensions>
    </extensions>
```
So basically I have an extension where the messages are compressed with gzip. I created my own GZipMessageEncodingElement.

And the other is a behavior extension where I have a  message inspector to validate some headers before the message is processed.

Appreciate so much.

Any clue?

**Comment by tecklee1967** [comment]
Hi @alexvazquez , have you found the solution? I am facing similar issue as well. 
Appreciate if you could share .. 

Thanks

**Reply by igor-nesterov-deltatre** [reply]
Are you able to add it programmatically?

**Reply by Panor520** [reply]
Hi @tecklee1967 ,did you solve the issue? 
I am facing similar issue as well.
Appreciate if you could share.
